lib/stack_master/commands/lint.rb

```diff
@@ -0,0 +1,42 @@
+require 'tempfile'
+
+module StackMaster
+  module Commands
+    class Lint
+      include Command
+      include Commander::UI
+
+      def initialize(config, stack_definition, options = {})
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def perform
+        unless cfn_lint_available
+          puts "Failed to run cfn-lint, do you have it installed and available in $PATH?"
+          return
+        end
+
+        Tempfile.open(['stack', '.json']) do |f|
```

Comment:
What if it's a YAML template?

Reply:
I was under the impression that proposed_stack.template_body will always be json. Is that incorrect?

If it can be yaml, that's supported too, just need to change the suffix.

Reply:
Yeah the template sent to CF is always JSON from StackMaster (even if the original template is YAML).

Reply:
AFAIK we support native YAML templates and we submit them as YAML to CloudFormation. I do not know though if template_body will always be json.